app/src/main/java/dev/arkbuilders/arkmemo/ui/adapters/TagAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package dev.arkbuilders.arkmemo.ui.adapters
```

Comment:
Tag feature is not yet implemented in MVP version, but might be there in post MVP.
So I'd like to keep this class around.